server/src/controllers/event.js

```diff
@@ -81,7 +81,9 @@ class EventController {
       { title, description, date, duration_min, max_participants }
     )
 
-    res.json({ success: true })
+    const event = await Event.findById(eventId)
+
+    res.json(event)
```

Comment:
Look grand, Thanks

Reply:
Can the number of the active events be sent on the trail object?
Also, a route for viewing events based on userID is required.

Reply:
Added: 

- Outing count on the route /trails. Use as `/trails?fields=outing_count`
- Join event: make a post request to `/trails/:trailId/events/:eventId/join`